pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -3738,6 +3738,9 @@ private Topic getTopicReference(TopicName topicName) {
         } catch (RestException e) {
             throw e;
         } catch (Exception e) {
+            if (e.getCause() == null){
```

Comment:
I think we should check if `e.getCause()` is null, when returns true throw the `e`, otherwise throw the`e.getCause()`.

Reply:
The root cause is ZK always timeout, so cannot get the topic, I still looking for a solution.

Reply:
+1

I've been thinking about why we encounter exceptions just now. thank you for the explanation.

Reply:
> The root cause is ZK always timeout, so cannot get the topic, I still looking for a solution.

@nodece
By the way, Could i help you with this issue ?

Reply:
Yeap, I created an issue for this: #13211, you can left a comment if have any ideas.